static/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -445,16 +446,42 @@ class SpanBar extends React.Component<SpanBarProps, SpanBarState> {
     errors: TraceError[] | null
   ) {
     const {generateContentSpanBarRef} = scrollbarManagerChildrenProps;
-    const {span, treeDepth} = this.props;
-
-    const operationName = getSpanOperation(span) ? (
-      <strong>
-        {getSpanOperation(span)}
-        {' \u2014 '}
-      </strong>
-    ) : (
-      ''
-    );
+    const {span, treeDepth, toggleSpanGroup} = this.props;
+
+    let titleFragments: React.ReactNode[] = [];
+
+    if (typeof toggleSpanGroup === 'function') {
+      titleFragments.push(
+        <Regroup
+          onClick={event => {
+            event.stopPropagation();
+            event.preventDefault();
+            toggleSpanGroup();
```

Comment:
Feels a little strange that you can ungroup by clicking the tree toggle but clicking on it once it's ungrouped collapses the spans.

Reply:
Yea I can see it being strange.

The API was developed before the UI was settled. 

I'll see if I can make this more clearer on the next PR revision.